### DISCUSSION TITLE
[Android App Update] - Possible to get 'assign barcode action' back on stock item?

### DISCUSSION THREAD

**Discussion by brechtdevuyst** [inventree/InvenTree#4993] [General]
I noticed that in the new UI update of the Android App (from 0.11.0 on, now running 0.12.1) there is no possibility anymore to scan and link a barcode anymore to a stock item. This is only possible if you receive a line item from a purchase order. Sometimes people forget to scan the QR-code or receive the line item on their pc, whith no qr-code scanner attached to, so the barcode still needs to be assigned after the item is already received from te purchase order.
But I can't seem to find that option/action anymore on a stock item. In the older versions (before 0.11.0) I'm certain it was there. 
Am I overlooking something, or did this option indeed got removed, and if so : would it be possible to bring it back?

Thanks in advance!
Kind regards

Brecht

**Comment by SchrodingersGat** [comment]
@brechtdevuyst this should be possible still in the latest version:

![image](https://github.com/inventree/InvenTree/assets/10080325/7f1baaea-cf99-467d-9702-aab359d29493)

![image](https://github.com/inventree/InvenTree/assets/10080325/906ecba7-46ca-49a3-84e0-12034c6cf93c)


What version of the back-end server are you running?

**Reply by brechtdevuyst** [reply]
Hi @SchrodingersGat, only the option 'scan into location' is visible, the 'assign barcode' not. 
I think you're right, the back-end server is only running 0.8.3. 
Would updating that solve the issue?

**Reply by SchrodingersGat** [reply]
@brechtdevuyst yep, the back-end is pretty outdated and updating would fix the app issue
